pkg/machine/e2e/machine_test.go

```diff
@@ -111,6 +111,9 @@ func setup() (string, *machineTestBuilder) {
 	if err := os.Unsetenv("SSH_AUTH_SOCK"); err != nil {
 		Fail("unable to unset SSH_AUTH_SOCK")
 	}
+	if err := os.Setenv("PODMAN_CONNECTIONS_CONF", filepath.Join(homeDir, "connections.json")); err != nil {
```

Comment:
In the interest of [allowing CI tests to run locally](https://issues.redhat.com/projects/RUN/issues/RUN-2126), would it make sense to set this to a tempdir?

Reply:
that is already a tmpdir, see above. The entire home is overwritten but as windows is using APPDATA it does not help there. Of course I could overwrite APPDATA for windows but this seems simpler and more consistent to me